assignment/src/com/assignmentweek2/person/Main.java

```diff
@@ -0,0 +1,18 @@
+package com.assignmentweek2.person;
+
+/*
+This is the main method
+ */
+public class Main {
+    public static void main(String[] args) {
+        Student Manoj = new Student("Manoj","Kurnool");
+        Doctor Ravi = new Doctor("Ravi","Hyderabad");
+        Teacher Suresh= new Teacher("Suresh","Nandyal");
```

Comment:
Variable names should start with small letters only.

Reply:
changed